src/app/Controllers/ErrorResult.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Microsoft.AspNetCore.Mvc;
```

Comment:
This should use the standard error handling which uses the IETF standard to return the error.